test/test.dart

```diff
@@ -532,26 +532,30 @@ extension on Map<String, String?> {
 
 BaasClient? _baasClient;
 Future<void> setupBaas() async {
-  final baasUrl = arguments[argBaasUrl];
-  if (baasUrl == null) {
-    return;
-  }
+  try {
+    final baasUrl = arguments[argBaasUrl];
+    if (baasUrl == null) {
+      return;
+    }
 
-  final cluster = arguments[argBaasCluster];
-  final apiKey = arguments[argBaasApiKey];
-  final privateApiKey = arguments[argBaasPrivateApiKey];
-  final projectId = arguments[argBaasProjectId];
-  final differentiator = arguments[argDifferentiator];
+    final cluster = arguments[argBaasCluster];
+    final apiKey = arguments[argBaasApiKey];
+    final privateApiKey = arguments[argBaasPrivateApiKey];
+    final projectId = arguments[argBaasProjectId];
+    final differentiator = arguments[argDifferentiator];
 
-  final client = await (cluster == null
-      ? BaasClient.docker(baasUrl, differentiator)
-      : BaasClient.atlas(baasUrl, cluster, apiKey!, privateApiKey!, projectId!, differentiator));
+    final client = await (cluster == null
+        ? BaasClient.docker(baasUrl, differentiator)
+        : BaasClient.atlas(baasUrl, cluster, apiKey!, privateApiKey!, projectId!, differentiator));
 
-  client.publicRSAKey = publicRSAKeyForJWTValidation;
+    client.publicRSAKey = publicRSAKeyForJWTValidation;
 
-  var apps = await client.getOrCreateApps();
-  baasApps.addAll(apps);
-  _baasClient = client;
+    var apps = await client.getOrCreateApps();
+    baasApps.addAll(apps);
+    _baasClient = client;
+  } catch (e) {
+    baasApps.clear();
```

Comment:
we are silencing all exceptions here. Not sure why we are doing that. Then it seems we allow the run to continue. What is the intention here?

Reply:
Right, I would've printed the exception. Yes, The tests continue to be run and all the local tests would pass and the sync tests will fail. This will cause the job to fail based on the test report. Otherwise, the tests are not ran and the job doesn't register any errors.